### ISSUE TITLE
Cannot parse the config file

### ISSUE THREAD

**Issue by MFry** [jsdoc3/jsdoc#1297]
I am running jsdoc on windows 7
```
$ jsdoc -v
JSDoc 3.4.3 (Thu, 10 Nov 2016 00:25:10 GMT)
```
and trying to run the documentations config file leads, which leads to the following:
```
$ jsdoc -c jsdocs.config
Cannot parse the config file jsdocs.config: SyntaxError: Unexpected token ﻿ in JSON at position 0
Exiting JSDoc because an error occurred. See the previous log messages for details.
```

The config file is the one used as an example on the site:
https://github.com/jsdoc3/jsdoc/blob/master/conf.json.EXAMPLE

**Comment by hegemonic** [comment]
I strongly suspect that you're saving the config file as UTF-8 with a BOM at the beginning. Try saving it in a different text encoding, or without the BOM.

**Comment by MFry** [comment]
**Explanation:** 
Windows tends to encode using [utf-8 with BOM](https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8) and you can check/modify the enconding with [notepad ](http://stackoverflow.com/a/6947786/1771644) or [notepad++](http://stackoverflow.com/a/39757642/1771644).
The issue is related to:
**Solution:**
Change to utf-8 without BOM for jsdoc. Take care with software such as visual studios which will change the format back by default when you hit save. 
**Reference**
[github.com/nodejs/1918](https://github.com/nodejs/node-v0.x-archive/issues/1918)
#1256

**Comment by johnfercher** [comment]
@MFry  , how can I convert to utf-8 without BOM in Linux 16.04?

**Comment by hegemonic** [comment]
FYI, this is now fixed on master. The fix will be included in JSDoc 3.5.0.
